tokio-timer/src/timer/mod.rs

```diff
@@ -62,18 +62,18 @@ use std::sync::atomic::AtomicUsize;
 use std::sync::atomic::Ordering::SeqCst;
 use std::usize;
 
-/// Timer implementation that drives [`Delay`], [`Interval`], and [`Deadline`].
+/// Timer implementation that drives [`Delay`], [`Interval`], and [`Timeout`].
 ///
 /// A `Timer` instance tracks the state necessary for managing time and
 /// notifying the [`Delay`] instances once their deadlines are reached.
 ///
 /// It is expected that a single `Timer` instance manages many individual
-/// `Delay` instances. The `Timer` implementation is thread-safe and, as such,
+/// [`Delay`] instances. The `Timer` implementation is thread-safe and, as such,
 /// is able to handle callers from across threads.
 ///
-/// Callers do not use `Timer` directly to create `Delay` instances.  Instead,
-/// [`Handle`] is used. A handle for the timer instance is obtained by calling
-/// [`handle`]. [`Handle`] is the type that implements `Clone` and is `Send +
+/// Callers do not use `Timer` directly to create [`Delay`] instances.  Instead,
+/// [`Handle`][Handle.struct] is used. A handle for the timer instance is obtained by calling
+/// [`handle`]. [`Handle`][Handle.struct] is the type that implements `Clone` and is `Send +
```

Comment:
githubs markdown is case insensitive, therefore `handle` and `Handle` should have different reference-links